<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>959</width>
    <height>679</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Gestionnaire de composants</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Chemin de la base de données : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_chooseDB">
       <property name="text">
        <string>Choisir</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_createDB">
       <property name="text">
        <string>Créer</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Double cliquer sur une cellule pour modifier sa valeur.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="editTriggers">
      <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>24</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>95</number>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Index</string>
      </property>
      <property name="toolTip">
       <string>Index</string>
      </property>
      <property name="textAlignment">
       <set>AlignLeading|AlignVCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Catégorie</string>
      </property>
      <property name="toolTip">
       <string>La catégorie à laquelle appartient le composant.</string>
      </property>
      <property name="textAlignment">
       <set>AlignLeading|AlignVCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Composant</string>
      </property>
      <property name="toolTip">
       <string>Le nm du composant.</string>
      </property>
      <property name="textAlignment">
       <set>AlignLeading|AlignVCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Section</string>
      </property>
      <property name="toolTip">
       <string>La forme de la section du composant. R pour Rectangulaire et C pour Cylindrique.</string>
      </property>
      <property name="textAlignment">
       <set>AlignLeading|AlignVCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Largeur (mm)</string>
      </property>
      <property name="toolTip">
       <string>Largeur de la section. Ou le diamètre pour une section cylindrique.</string>
      </property>
      <property name="textAlignment">
       <set>AlignLeading|AlignVCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Épaisseur (mm)</string>
      </property>
      <property name="toolTip">
       <string>Épaisseur de la section. Ou le diamètre pour une section cylindrique.</string>
      </property>
      <property name="textAlignment">
       <set>AlignLeading|AlignVCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Longueur (mm)</string>
      </property>
      <property name="toolTip">
       <string>Longueur du composant, laisser à 0 si ce composant doit s'adapter aux dimensions du produit lors de la création.</string>
      </property>
      <property name="textAlignment">
       <set>AlignLeading|AlignVCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Couleur</string>
      </property>
      <property name="toolTip">
       <string>La couleur de la forme.</string>
      </property>
      <property name="textAlignment">
       <set>AlignLeading|AlignVCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Masse volumique (kg/m³)</string>
      </property>
      <property name="toolTip">
       <string>La masse volumique du matériau utilisé.</string>
      </property>
      <property name="textAlignment">
       <set>AlignLeading|AlignVCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>CAO</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonAdd">
       <property name="toolTip">
        <string>Cliquer ici pour ajouter un composant, cela ajoutera une ligne qu'il faudra remplir.</string>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonDuplicate">
       <property name="text">
        <string>Dupliquer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonDel">
       <property name="toolTip">
        <string>Cliquer ici pour supprimer un composant. La ligne sélectionnée sera supprimée.</string>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonRej">
       <property name="toolTip">
        <string>Annule tous les changements effectué et ferme la fenêtre.</string>
       </property>
       <property name="text">
        <string>Annuler</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonVal">
       <property name="toolTip">
        <string>Valide tous les changements et quitte la fenêtre.</string>
       </property>
       <property name="text">
        <string>Valider</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
